autosklearn/pipeline/components/data_preprocessing/text_encoding/__init__.py

```diff
@@ -0,0 +1,114 @@
+from collections import OrderedDict
+import os
+
+from typing import Any, Dict, Optional
+
+from ConfigSpace import Configuration
+from ConfigSpace.configuration_space import ConfigurationSpace
+from ConfigSpace.hyperparameters import CategoricalHyperparameter
+
+from sklearn.base import BaseEstimator
+
+from ...base import AutoSklearnPreprocessingAlgorithm, find_components, \
+    ThirdPartyComponents, AutoSklearnChoice
+
+from autosklearn.pipeline.base import DATASET_PROPERTIES_TYPE, PIPELINE_DATA_DTYPE
+
+bow_directory = os.path.split(__file__)[0]
+_bows = find_components(__package__,
+                        bow_directory,
+                        AutoSklearnPreprocessingAlgorithm)
+_addons = ThirdPartyComponents(AutoSklearnPreprocessingAlgorithm)
```

Comment:
@mfeurer Seems like this is conflicting with [this](https://github.com/automl/auto-sklearn/blob/development/autosklearn/pipeline/components/classification/__init__.py#L16) which was done to solve some multi-processing things in a more recent development push.

```python
additional_components = ThirdPartyComponents(AutoSklearnClassificationAlgorithm)
_addons['classification'] = additional_components


def add_classifier(classifier: Type[AutoSklearnClassificationAlgorithm]) -> None:
    additional_components.add_component(classifier)
```

Recommended change
```python
additional_components = ThirdPartyComponents(AutoSklearnPreprocessingAlgorithm)
_addons['bow'] = additional_components


def add_bow(classifier: Type[AutoSklearnClassificationAlgorithm]) -> None:
    additional_components.add_component(classifier)
```

Reply:
Do i have to update my forked reposetory in anyway to make that work ? Just applying the changes yields many errors. When did you implemented the  multi-processing things ?